PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -251,8 +250,11 @@ public void finished(boolean aborted, int errors, int warnings, CompileContext c
                                         null,
                                         postEventProperty);
 
-                                //FIXME!!! Sometimes, the submission message window not ready
-                                //   HDInsightUtil.showInfoOnSubmissionMessageWindow(project, String.format("Info : Build %s successfully.", artifact.getOutputFile()));
+                                DefaultLoader.getIdeHelper().executeOnPooledThread(() -> {
```

Comment:
DefaultLoader.getIdeHelper().executeOnPooledThread is not necessary here since we will make sure it will be execute on UI thread.